### DISCUSSION TITLE
Release v1.4.0.0

### DISCUSSION THREAD

**Discussion by mday-dwa** [dreamworksanimation/openmoonray#130] [Announcements]
### Build Improvements
* Upgraded to Embree 4.1.0
* Upgraded to jsoncpp 1.9.5
* Updated Centos 7 instructions to match procedure for Rocky Linux 9
### Features
* Added telemetry overlay functionality
* Added GPU mode for denoising via OpenImageDenoise
* XPU mode becomes the auto and default rendering modes
* Further improvements to rdl2_print
### Bug Fixes
* Fixed frame locked noise on volumes
* Fixed data rate for user data with identical counts
* Fixed handling of missing materials for DwaLayerMaterial
* Fixed extremely slow processing of large meshes with ngons
* Fixed redundant log messages
* Fixed ImageMap crash when updating delta file
* Fixed compilation errors, logging issues, and threaded logging crash with new shader logging framework
* Fixed NVidia driver version check & requirement
* Fixed alpha channel in volume passes with presence on cutouts
* Fixed xpu fallback behavior
* Fixed fallback logic for refractive cryptomattes in vector mode
* Fixed CombineLightFilter bug causing crash
* Fixed CookieLightFilter vector, debug mode bug
* Fixed hair and hair diffuse lobe labels
* Fixed occasional black pixels caused by out-of-range density attribute on LightFilters
* Fixed addition of explicit attributes for unassigned parts
* Fixed crash in vector mode caused by ray termination lights
* Fixed data rate on UserData based on set USD interpolation
* Fixed delta slowdown due to inefficient log outputs
* Fixed per part user data when only the mesh name is specified
* Fixed expected component count for varying rate primitive attributes on RdlMeshGeometry
* Fixed visibility aov
* Multi-machine rendering: Fixed packTiles codec when decoding an empty image
* Multi-machine rendering: Fixed progressiveFrame logic when decoding an empty image block
* Fixed minicoord core assignment issue
* render_profile_viewer: Fixed crash when refitting render prep stats
* Miscellaneous build fixes
### Other
* Improved comments for LightSet and ShadowSet attributes
* Improved comments for ImageMap
* Added further scene variable documentation
* Multi-machine rendering: Added new Morton_shiftflip tile scheduler
* Multi-machine rendering: Added dynamic checkpoint interval adjusting logic for multi-machine configuration
* Added robust snapshotDelta unitTest
* Removed incomplete/unsupported multiple camera support
* hdMoonray now allows UsdSphereLight to be a MoonRay SpotLight

<hr /><em>This discussion was created from the release <a href='https://github.com/dreamworksanimation/openmoonray/releases/tag/openmoonray-1.4.0.0'>Release v1.4.0.0</a>.</em>

**Comment by rgwilson68** [comment]
The build documentation has been updated for this release. 

- The new Centos7 directory contains build instructions for Centos-7, updated to match the Rocky9 instructions.
- The new file building/general_build.md has more detailed information about the general build process, which should help to answer some questions about how adapt the build to different requirements

These updates have also been pushed to the documentation site : see https://docs.openmoonray.org/getting-started/installation/building-moonray/

**Comment by rgwilson68** [comment]
NB: we accidentally left the files for the old Centos 7 build in place, directly under the /building directory. These don't reflect the latest documentation : the files corresponding to the new documentation are in /building/centos7

**Comment by JeepLambo9** [comment]
how do i download it
